### DISCUSSION TITLE
[Vote ended on 2022-10-11] Improve licensing rules for 'all other code' in the collection inclusion requirements

### DISCUSSION THREAD

**Discussion by felixfontein** [ansible-community/community-topics#142] [Votes done]
Should the collection inclusion rules be updated to explicitly mention that code outside plugins/ can have other licenses? More precisely: should the update https://github.com/ansible-collections/overview/pull/212 to the rules be merged?

Discussion topic: #123

Please vote with a +1 to merge https://github.com/ansible-collections/overview/pull/212 or -1 to not merge it.

**Comment by felixfontein** [comment]
+1

**Comment by briantist** [comment]
+1

**Comment by mariolenz** [comment]
IANAL but as far as I understand this: +1

**Comment by markuman** [comment]
+1

**Comment by gotmax23** [comment]
+1

**Comment by Andersson007** [comment]
+1

**Comment by gundalow** [comment]
+1

**Comment by acozine** [comment]
+1, a good clarification/explanation of the existing rules.

**Comment by tremble** [comment]
+1 we should be as accurate as possible about anything licensing related.

**Comment by russoz** [comment]
+1
